src/qibolab/platforms/abstract.py

```diff
@@ -152,6 +159,140 @@ def reload_settings(self):
             else:
                 self.qubits[q] = Qubit(q, **settings["characterization"]["single_qubit"][q])
 
+    def dump(self, path: Path):
+        with open(path, "w") as file:
+            yaml.dump(self.settings, file, sort_keys=False, indent=4, default_flow_style=None)
+
+    def update(self, updates: dict):
+        r"""Updates platform common runcard parameters after calibration actions.
+
+        Args:
+
+            updates (dict): Dictionary containing the parameters to update the runcard.
+        """
+
+        for par, values in updates.items():
+            for qubit, value in values.items():
+                # resonator_spectroscopy / resonator_spectroscopy_flux / resonator_punchout_attenuation
+                if par == "readout_frequency":
+                    freq = int(value * 1e9)
```

Comment:
Why are we converting from GHz to Hz? I thought we were using Hz everywhere

Reply:
Good point. @andrea-pasquale @Edoardo-Pedicillo does the autocalibration return the value in GHz?

Reply:
The fitting is now performed considering GHz and not Hz to improve the stability, also in the plotting we show GHz and not BHz as it was happening before which was not ideal. This is why we are currently using `GHz`.
We could also convert it back to Hz just for the runcard update mechanism. Let me know what you think.

Reply:
Either solution is fine with me, as long as it is documented that `platform.update` expects the frequencies in GHz.